src/felzenszwalb.jl

```diff
@@ -9,73 +9,55 @@ The function returns the number of segments and index mapping from nodes of the
 Parameters:  
 -    edges          = Array of edges in RAG. Each edge is represented as `ImageEdge`.
 -    num_vertices   = Number of vertices in RAG
--    k              = Threshold for region splitting step. Larger threshold will result in bigger segments.
+-    k              = Threshold for region merging step. Larger threshold will result in bigger segments.
+-    min_size       = Minimum segment size 
 
 """
-function felzenszwalb_graph(edges::Array{ImageEdge}, num_vertices::Int, k::Real)
+function felzenszwalb_graph(edges::Array{ImageEdge}, num_vertices::Int, k::Real, min_size::Int = 0)
```

Comment:
Seems like this too could be called `felzenszwalb`, since it will dispatch to this version whenever you supply an `Array{ImageEdge}`.